packages/tabs/src/Tab.js

```diff
@@ -6,6 +6,9 @@ import TabPresenter from "./presenters/TabPresenter";
 export default class Tab extends Component {
   static propTypes = {
     /**
+     * DEPRECATED: Use the activeTab or defaultActiveTabIndex prop on the Tabs
+     * component instead.
+     *
```

Comment:
Thought we can just remove `active` from propTypes/defaultProps for `Tab`, just like the other props (`onMouseLeave`,`showDivider`, etc. ) that not open to end users, but will be overwritten via payloads from parent's `renderTabs` function. And just deconstruct it from `...otherProps` when needed in the render function?

Reply:
I think we can leave `active` for now to let the users know that it is on the road to being deprecated.  For the first line,  `DEPRECATED: Use the activeTab or defaultActiveTabIndex prop on the Tabs...` the actual prop is `activeTabIndex` and not `activeTab`?